liquibase-core/src/main/java/liquibase/parser/core/xml/XMLChangeLogSAXParser.java

```diff
@@ -150,6 +151,13 @@ public void fatalError(SAXParseException exception) throws SAXException {
             }
 
             throw new ChangeLogParseException("Invalid Migration File: " + reason, e);
+        } catch (InvalidPathException e) {
+            final String message = "ERROR: The file '" + e.getInput() + "' was not found. " +
+                    "The <changelog|snapshot> file property cannot be configured with " +
+                    "a fully qualified path, but must be a relative path on the property, " +
+                    "and any local or remote base of the path set on the '--search-path' property.";
+            Scope.getCurrentScope().getUI().sendErrorMessage("ERROR: " + e.getMessage(), e);
```

Comment:
This is probably more appropriate in the logs than in the console, since the exception you're throwing on line 160 will show up in the console.

```suggestion
            Scope.getCurrentScope().getLog(getClass()).error("ERROR: " + e.getMessage(), e);
```

Reply:
good point!

Reply:
I think there is no `error` method, I am going to use `severe` then.